src/makeCalculator.js

```diff
@@ -37,7 +37,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(num) {
+      this.result += num;
```

Comment:
Make it like a pure function.
Let this method return you a pure result, and save it in your operate method.
For example
```
add(num1, num2) {
	return num1 + num2;
}